### ISSUE TITLE
Mayor on 'Checkins' doesn't have matching vertical padding on bottom

### ISSUE THREAD

**Issue by GoogleCodeExporter** [iamakshayv/foursquared#78]
```
The user photo has a few pixels above the top (separating it from the 'Mayor' 
titlebar).

But, the bottom is flush with 'Recent Checkins' titlebar. Very obvious when 
highlighted.

Minor UI nitpick.
```

Original issue reported on code.google.com by `devin.re...@gmail.com` on 30 Aug 2009 at 8:29

**Comment by GoogleCodeExporter** [comment]
Original comment by `jlapenna` on 31 Aug 2009 at 1:56

**Comment by GoogleCodeExporter** [comment]
Original comment by `jlapenna` on 31 Aug 2009 at 2:06
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision ebbd75b4f6.
```

Original comment by `jlapenna` on 5 Jan 2010 at 5:47
